Dockerfile

```diff
@@ -29,7 +32,9 @@ RUN apt-get update && \
 WORKDIR /m2cgen
 
 COPY requirements-test.txt ./
-RUN pip3 install --no-cache-dir Cython && \
+RUN update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.7 1 && \
+    pip3 install --upgrade pip && \
+    pip3 install --no-cache-dir Cython numpy && \
```

Comment:
`numpy` is being used as part of a `setup.py` in either statsmodels or lighting (don't remember which one, can double check if needed), that's why we adding it here.